project.clj

```diff
@@ -30,7 +30,6 @@
             [lein-pprint "1.1.1"]
             [lein-ring "0.9.7"]]
   :main datomic-rest-api.get-handler
```

Comment:
Think we still need to specify [skip-aot](mount/src/mount/) for the `:main` namespace, e.g:

```
:main ^:skip-aot  datomic-to-catalyst.get-handler
```

Additionally, I think we want to preserve aot for uberjar builds.
This can be done by adding:
```
:uberjar {:aot :all}
```
to the `:profiles` map.